src/test/java/wooteco/subway/path/application/PathServiceTest.java

```diff
@@ -0,0 +1,33 @@
+package wooteco.subway.path.application;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.transaction.annotation.Transactional;
+import wooteco.subway.member.domain.LoginMember;
+import wooteco.subway.path.dto.PathResponse;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest
+@Transactional
+public class PathServiceTest {
+
+    private final PathService pathService;
+
+    @Autowired
+    public PathServiceTest(PathService pathService) {
+        this.pathService = pathService;
+    }
```

Comment:
```suggestion
    @Autowired
    private final PathService pathService;
```

테스트에서는 보통 이렇게 작성해요. 테스트 특성 상 주입 받을 bean이 많아지면 필드 주입이 편하고 가독성도 좋아서요! :)